.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
See https://blog.travis-ci.com/2016-04-07-migration-update and https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments